v2/internal/reconcilers/arm/azure_generic_arm_reconciler.go

```diff
@@ -91,28 +92,35 @@ func (r *AzureDeploymentReconciler) asARMObj(obj genruntime.MetaObject) (genrunt
 	return typedObj, nil
 }
 
-func (r *AzureDeploymentReconciler) makeInstance(log logr.Logger, eventRecorder record.EventRecorder, obj genruntime.MetaObject) (*azureDeploymentReconcilerInstance, error) {
+func (r *AzureDeploymentReconciler) makeInstance(ctx context.Context, log logr.Logger, eventRecorder record.EventRecorder, obj genruntime.MetaObject) (*azureDeploymentReconcilerInstance, error) {
 	typedObj, err := r.asARMObj(obj)
 	if err != nil {
 		return nil, err
 	}
 	// Augment Log with ARM specific stuff
 	log = log.WithValues("azureName", typedObj.AzureName())
 
+	armClient, credentialFrom, err := r.ARMClientFactory(ctx, typedObj)
+	if err != nil {
+		return nil, err
+	}
+
+	eventRecorder.Eventf(obj, v1.EventTypeNormal, "credentialFrom", "Using credential from %q", credentialFrom)
```

Comment:
```suggestion
	eventRecorder.Eventf(obj, v1.EventTypeNormal, "CredentialFrom", "Using credential from %q", credentialFrom)
```

I think our one other event is uppercase currently and I took a quick look at Kubernetes documentation and it showed them using Uppercase events as well.

Reply:
I saw this one, k8s automagically makes it upper case. But yeah, worth making it upper case here